f8a_worker/workers/dependency_parser.py

```diff
@@ -127,14 +127,15 @@ def get_npm_dependencies(cls, path):
                                           package=name, version=version))
 
                 if transitive_deps:
```

Comment:
This check is redundant as the following `for` statement handles the existence of the array.

Reply:
It will not, since we get `None` if transitive dependencies are not present and if you directly iterate a `None` object you will get `TypeError: 'NoneType' object is not iterable`

Reply:
But, we can let the `get` return a default empty list and then let for loop take care of it.

Reply:
@abs51295 The above is a cleaner approach.